include/RAJA/LegacyCompatibility.hxx

```diff
@@ -0,0 +1,215 @@
+#ifndef KRIPKE_LEGAGY_COMPATIBILITY_HXX
+#define KRIPKE_LEGAGY_COMPATIBILITY_HXX 1
+
+static_assert(__cplusplus >= 201103L, "C++ standards below 2011 are not supported");
+
+#include <type_traits>
+#include <utility>
+#include <iostream>
+#include <functional>
+#include <cstdint>
+
+namespace VarOps {
```

Comment:
I feel that impl has a certain association in c++11, I think it's being used correctly here, so I'd just use it, personally. Though if we're going with facetious suggestions, I suggest raja::openmp::kokkos::cuda::openacc::chapel::agency::fiddling_with_magnets